cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -2721,7 +2601,7 @@ void CacheAllocator<CacheTrait>::evictForSlabRelease(
 
 template <typename CacheTrait>
 typename CacheAllocator<CacheTrait>::WriteHandle
-CacheAllocator<CacheTrait>::evictNormalItemForSlabRelease(Item& item) {
+CacheAllocator<CacheTrait>::evictNormalItem(Item& item) {
```

Comment:
In L2620, do we still need `accessContainer->remoevIf` to return us a handle?

It seems we just need to return `true/false`.